### PR TITLE
Update to gRPC Swift 1.0.0 alpha.16

### PR DESCRIPTION
**Pull request by abrampers**

### CODE REVIEW

**Comment by vyshane** [comment]
Thanks for the PR @abrampers. I will take a look within the next 24 hours 👍

**Comment by vyshane** [comment]
I would rather we not include protoc and the protoc-gen-swift, protoc-gen-grpc-swift plugins directly in this repo.

File: Makefile [hunk 1]
```diff
@@ -1,18 +1,22 @@
 PROTO_DIR = Tests/Protobuf
 PROTO_GEN_DIR = Tests/CombineGRPCTests/Generated
+PROTOC_PATH= protoc/bin/protoc
```
Comment (vyshane) [inline Makefile:3]: Sorry, I don't think it's a good idea to add binaries of unknown version/provenance into the repo. Can you remove protoc and plugins?

Comment (abrampers) [reply Makefile]: Sure. I'll delete it right away. 

But in my opinion, I think it is better to use binaries included from the repo to compile the `.proto` files to reduce build error if the local `protoc` compiler version is not correct.

**Pull request merged**